_includes/homepage/hero/components/hero-infobox-desktop.html

```diff
@@ -0,0 +1,24 @@
+<div
+  class="p16 is-hidden-mobile"
+  style="width: {{- section.hero.size -}}; background: {% if section.hero.backgroundColor == 'gray' %} #00000080 {% else %} {{- section.hero.backgroundColor -}} {% endif %};"
+>
+  <div class="py16">
+    <div class="mb8">
+      <h1 class="hero-title mb4">
+        <b class="is-hidden-touch">{{- section.hero.title -}}</b>
+        <b class="is-hidden-desktop">{{- section.hero.title -}}</b>
```

Comment:
Is this supposed to be here twice? What happens for non touch and desktop layouts (e.g. tablet, extra wide)?

Reply:
this was actually copied as-is from the existing code. that being said, they seem to cover all viewports so actually i can just remove one and set it as `b` without any classes it seems like
![Screenshot 2023-09-20 at 4 11 01 PM](https://github.com/isomerpages/isomerpages-template/assets/44049504/59a23636-e994-4186-bf95-87273b142fe9)